File: tests/fixtures/csharp/hello.cs
using System;
using System.Collections.Generic;

public class Calculator
{
    public int Add(int a, int b)
    {
        return a + b;
    }

    public void Print(string message)
    {
        Console.WriteLine(message);
    }
}

public struct Point
{
    public double X;
    public double Y;
}

public interface IProcessor
{
    void Process(string data);
}

public enum Color
{
    Red,
    Green,
    Blue
}

File: tests/fixtures/swift/hello.swift
import Foundation

class Calculator {
    func add(a: Int, b: Int) -> Int {
        return a + b
    }

    func print(message: String) {
        print(message)
    }
}

protocol IProcessor {
    func process(data: String)
}

enum Color {
    case red
    case green
    case blue
}

struct Point {
    let x: Double
    let y: Double
}

typealias StringMap = Dictionary<String, String>
